openreview/arr/webfield/programChairsWebfield.js

```diff
@@ -146,6 +146,193 @@ return {
       })
       return checklistReplies?.length??0;
       `
-    }    
+    },
+    reviewerEmailFuncs: [
+      {
+        label: 'Reviewers with Unreplied Rebuttals', filterFunc: `
+        console.log(row);
+        if (row.notesInfo.length <= 0){
+          return false;
+        }
+
+        return row.notesInfo.some(obj => {
+          const reviewId = obj?.officialReview?.id ?? ''
+          const anonId = obj?.officialReview?.anonymousId ?? ''
+          const replies = obj?.note?.details?.replies ?? []
+          const authorReplies = replies.filter(reply => 
+            reply.replyto === reviewId && reply.signatures.some(sig => sig.includes('Authors'))
+          )
+          console.log(replies)
+          console.log(authorReplies)
+          const revRepliesToAuthorReplies = authorReplies.map(authorReply => 
+            replies.filter(reply => reply.replyto === authorReply.id && reply.signatures.some(sig => sig.includes(anonId)))
+          ).flat()
+
+          if (reviewId.length <= 0) {
+            return false
+          }
+
+          if (reviewId.length > 0 && revRepliesToAuthorReplies.length < authorReplies.length) {
+            return true
+          }
+
+        })
+        `
+      },
+      {
+        label: 'Registered Reviewers with Unsubmitted Load', filterFunc: `
+        const registrationNotes = row.reviewerProfile?.registrationNotes ?? []
+        if (registrationNotes.length <= 0) {
+          return false
+        }
+
+        const registrationForm = registrationNotes.filter(note => {
+          const invitations = note?.invitations ?? []
+          return invitations.some(inv => inv.includes('Reviewers/-/Registration'))
+        })
+        const maxLoadForm = registrationNotes.filter(note => {
+          const invitations = note?.invitations ?? []
+          return invitations.some(inv => inv.includes('Reviewers/-/Max_Load_And_Unavailability_Request'))
+        })
+
+        if (registrationForm.length >= 1 && maxLoadForm.length <= 0) {
+          return true
+        }
+        return false
+        `
+      },
+      {
+        label: 'Registered Reviewers with Zero Load', filterFunc: `
+        const registrationNotes = row.reviewerProfile?.registrationNotes ?? []
+        if (registrationNotes.length <= 0) {
+          return false
+        }
+
+        const registrationForm = registrationNotes.filter(note => {
+          const invitations = note?.invitations ?? []
+          return invitations.some(inv => inv.includes('Reviewers/-/Registration'))
+        })
+        const maxLoadForm = registrationNotes.filter(note => {
+          const invitations = note?.invitations ?? []
+          return invitations.some(inv => inv.includes('Reviewers/-/Max_Load_And_Unavailability_Request'))
+        })
+        
+        if (registrationForm.length <= 0 || maxLoadForm.length <= 0) {
```

Comment:
if a reviewer did not submit the registration note, but they selected 0 in "maximum_load_this_cycle", they would not be emailed?

Reply:
I understand that registration notes are always present but I don't know why we need to check if they exist.

Also I think reminder should be sent when there is no reply for Max_Load_And_Unavailability_Request not when the load is 0, right?

Reply:
Yeah I think I can remove this option, it was used one cycle to get more capacity out of the reviewers but there's probably a better way to do this. Also, this option is not on the list of email options requested.